### DISCUSSION TITLE
Email is "stuck" in signup and can't be used

### DISCUSSION THREAD

**Discussion by adinklotz** [vercel/community#6094] [Account Issue] [answered]
### Question

I tried to create an account with Sign up with Email. However, something seems to have gone wrong with the email verification process - when I try to log in I get the error "We are unable to complete your signup". So I tried creating an account signing in with my Github account, but now it won't let me change my account email to my desired email, since it's apparently tied up somewhere trying to register my account. Is there a way we can cancel that registration and make my email usable again?

**Comment by swarnava** [comment, marked as answer]
Hi,

Please email us at registration@vercel.com from your registered email.
